verification/src/contextual_block_verifier.rs

```diff
@@ -471,21 +513,41 @@ impl<'a, CS: ChainStore<'a>> ContextualBlockVerifier<'a, CS> {
             prepare_epoch_ext(&self.context, &parent)?
         };
 
```

Comment:
Do we need to verify `header number == parent number + 1` in ContextialBlockVerifier?

Reply:
No
HeaderVerifier -> BlockVerifier(non-contextial) -> ContextialBlockVerifier 
this order must be guarantee.